### ISSUE TITLE
Could not download Node.js: Got error code 404 from the server.

### ISSUE THREAD

**Issue by emeraldhieu** [eirslett/frontend-maven-plugin#1054]
**Do you want to request a _feature_ or report a _bug_?**

bug

**What is the current behavior?**

This error is displayed
[INFO] Installing node version v12.13.1
[INFO] Downloading https://nodejs.org/dist/v12.13.1/node-v12.13.1-darwin-arm64.tar.gz to /Users/nghi/.m2/repository/com/github/eirslett/node/12.13.1/node-12.13.1-darwin-arm64.tar.gz
[INFO] No proxies configured
[INFO] No proxy was configured, downloading directly

**If the current behavior is a bug, please provide the steps to reproduce.**

Instlal JHipster's gateway on Apple M1

**What is the expected behavior?**

Should install successfully

**Please mention your frontend-maven-plugin and operating system version.**

v12.13.1, Mac Apple M1

**Comment by eirslett** [comment]
This bug is here because Node 12 was released before the M1 chip, so there are no arm builds for it. Can you upgrade to Node 16 or 18?
I'll close this issue because Node 12 has reached end-of-life.

**Comment by moeriki** [comment]
Yes but 14 has not reached EOL.

nvm, if there is arm64 build, falls back to x64. Which runs fine.

```sh
# nvm install 14
Installing Node v14.21.3 lts/fermium
Fetching https://nodejs.org/dist/v14.21.3/node-v14.21.3-darwin-x64.tar.gz
Now using Node v14.21.3 (npm 6.14.18) ~/.local/share/nvm/v14.21.3/bin/node
```

```sh
# nvm install 20
Installing Node v20.3.0 latest
Fetching https://nodejs.org/dist/v20.3.0/node-v20.3.0-darwin-arm64.tar.gz
Now using Node v20.3.0 (npm 9.6.7) ~/.local/share/nvm/v20.3.0/bin/node
```

Is this something worth considering?

**Comment by davidarroyo1234** [comment]
Same issue here, fixed with frontend-maven-plugin  +1.11.3 thanks @eirslett
